torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -545,6 +545,35 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                             args[0],
                             delete_user_cb=lambda node: node != args[0],
                         )
+            if node.target == torch.ops.aten.slice.Tensor:
+                # Returns the new node or `None` if unsupported
+                def callback(tensor, dim, start, end, step=1):
+                    input_size = tensor.meta["val"].size()
+
+                    if step != 1 or dim >= len(input_size):
+                        return None
+
+                    # NOTE(jdh8): is there a way to skip a no-op?
+                    # if start == 0 and end >= input_size[dim]:
+                    #    return tensor
+
+                    if len(input_size) != 4:
+                        return None
+
+                    slice_start = np.zeros(len(input_size), dtype=int)
+                    slice_end = np.array(input_size)
+
+                    slice_start[dim] = start
+                    slice_end[dim] = min(end, input_size[dim])
+                    slice_end -= 1
```

Comment:
thats because torch is exclusive and ttnn is inclusive, right?

Reply:
I am going to refactor ttnn.slice to be exclusive